lib/editor-linter.js

```diff
@@ -22,9 +22,12 @@ export default class EditorLinter {
     this.subscriptions.add(this.editor.onDidDestroy(() =>
       this.dispose(),
     ))
-    this.subscriptions.add(this.editor.onDidSave(debounce(() =>
-      this.emitter.emit('should-lint', false),
-    ), 16, true))
+    this.subscriptions.add(this.editor.onDidSave(
+      debounce(() => this.emitter.emit('should-lint', false), 16, true),
+    ))
+    this.subscriptions.add(this.editor.getBuffer().onDidReload(
+      debounce(() => this.emitter.emit('should-lint', false), 16, true),
```

Comment:
Can we change the debounce parameters to `1000` and `false`, so it does not invoke until onDidReload stops invoking, and calls the callback 1 second after the last call